onboard/cv/main.cpp

```diff
@@ -117,7 +117,7 @@ int main() {
       //update both tags in LCM message
       //first tag
       if(tagPair.first.id == -1){//no tag found
-        if(left_tag_buffer <= 5){//send the buffered tag
+        if(left_tag_buffer <= 20){//send the buffered tag
```

Comment:
Why did you choose 20?

Reply:
20 was chosen arbitrarily- we want to test with auton on the rover before determining a final number.